packages/lib/service/credential/getUserCredential.ts

```diff
@@ -0,0 +1,51 @@
+import logger from "@calcom/lib/logger";
+
+const log = logger.getSubLogger({ prefix: ["getCredential"] });
+
+export const getUserCredential = ({
```

Comment:
A user credential in case of Delegation Credential is basically a combination of `delegationCredentialId` and `userId` while for a regular credential it is just `credentialId`